NDXINTER/reduce.py

```diff
@@ -1,41 +1,171 @@
-import os
-import shutil
 import sys
-sys.path.append("/isis/NDXMUSR/user/scripts/autoreduction")
+
+sys.path.append('/opt/Mantid/scripts')
+sys.path.append('/opt/Mantid/scripts/SANS')
+sys.path.append('/opt/Mantid/lib')
+sys.path.append('/opt/Mantid/scripts/Inelastic')
+sys.path.append('/opt/Mantid/scripts/Engineering')
+sys.path.append('/opt/Mantid/scripts/Interface')
+sys.path.append('/opt/Mantid/scripts/Diffraction')

```

Comment:
Shouldn't need those anymore - they now get [appended in the container](https://github.com/ISISScientificComputing/autoreduce-containers/blob/master/qp_singularity_wrap.def#L8)